css/main.css

```diff
@@ -0,0 +1,50 @@
+.overlay {
+    /* Behave as a frame for overlay contents */
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+
+    /* Use CSS Grid for alignment/placement of contents */
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+    grid-template-rows: auto 50px 1fr 20px auto;
+    grid-template-areas:
+        " title     .     hideBtn"
+        "   .       .        .   "
+        "   .       .      tool  "
+        "   .       .      tool  "
+        "control    .        .   ";
+}
```

Comment:
Warning: grid isn't supported on Internet Explorer!!!
Oh wait nvm no one (not even Github) cares about supporting IE, so this is fine.